cherry_picker/test_cherry_picker.py

```diff
@@ -558,6 +559,22 @@ def test_normalize_short_commit_message():
     )
 
 
+def test_remove_commit_prefix():
+    assert (
+        remove_commit_prefix("[3.12] Fix something (GH-3113)")
+        == "Fix something (GH-3113)"
+    )
+    assert (
+        remove_commit_prefix("[3.11] [3.12] Fix something (GH-3113)")
+        == "Fix something (GH-3113)"
+    )
+    assert remove_commit_prefix("Fix something (GH-3113)") == "Fix something (GH-3113)"
+    assert (
+        remove_commit_prefix("[WIP] Fix something (GH-3113)")
+        == "[WIP] Fix something (GH-3113)"
+    )
```

Comment:
Let's `@pytest.mark.parametrize` so each test case is run in isolation. Something like:

```suggestion
@pytest.mark.parametrize(
    "commit_message, expected",
    [
        ("[3.12] Fix something (GH-3113)", "Fix something (GH-3113)"),
        ("[3.11] [3.12] Fix something (GH-3113)", "Fix something (GH-3113)"),
        ("Fix something (GH-3113)", "Fix something (GH-3113)"),
        ("[WIP] Fix something (GH-3113)", "[WIP] Fix something (GH-3113)"),
    ],
)
def test_remove_commit_prefix(commit_message, expected):
    assert remove_commit_prefix(commit_message) == expected
```